### ISSUE TITLE
Cannot pass basedir when using CLI

### ISSUE THREAD

**Issue by StreetStrider** [pugjs/jade#2236]
Hello. `basedir` option is required to resolve «absolute» includes. It works when I'm using Jade via JS API, but not works when I'm using CLI. Looks like CLI isn't passing any `--basedir` down to options and to `resolvePath`.

**Comment by TimothyGu** [comment]
`--basedir` isn't an option. Use `-O '{"basedir": "<bleh>"}'`

**Comment by StreetStrider** [comment]
@TimothyGu, thanks, I've simply switched to JS API for now, it is OK for my pipeline. Any reason why `--basedir` shouldn't be implemented directly?

**Comment by ForbesLindesay** [comment]
I don't see any reason why it can't be added.  Please open an issue on the [cli repo](https://github.com/pugjs/pug-cli) if you want it :+1:

**Comment by StreetStrider** [comment]
Thanks, I'll adress my CLI-related issues and PRs there, then. One question. Why the org is named Pug? I don't know any Pug JS, I know Jade.

**Comment by ForbesLindesay** [comment]
See #2184

**Comment by 9nix00** [comment]
interesting, I fix it too.  
 #pugjs/pug-cli/18

this is very very useful when we need add script in jade/pug command in package.json.
if you want add script in your package.json, you have to escape quote.
``` javascript

    "jade": "jade  -E jinja --obj {\"basedir\":\\\"../\\\"} ../",

```

 it looks very very ugly! am I  right ?

**Comment by StreetStrider** [comment]
@ForbesLindesay, @TimothyGu there's no activity at CLI repo for two weeks, so I ping here. :bell:

**Comment by chharvey** [comment]
@9nix00 THANK YOU I've been trying to figure out how to escape the quotes properly. Why *three* back slashes??

**Comment by TimothyGu** [comment]
@chharvey, the first two escapes becomes one backslash after JSON is parsed, and the third one escapes the quotation mark. Then the one backslash escapes the quotation mark when the shell is invoked.

I'll merge the CLI change soon.

**Comment by TimothyGu** [comment]
Fixed in pug-cli@1.0.0-alpha6.
